docs/application-insights.md

```diff
@@ -0,0 +1,14 @@
+- [Application Insights](#application-insights)
```

Comment:
I don't think that this link is needed anymore in this file as there's only one section in it